queue/util/createConsumer.js

```diff
@@ -87,8 +65,10 @@ const createConsumer = module.exports = async function (options) {
   })
 
   amqConnection.on('error', _handleConnectionClosed)
-  amqConnection.on('close', _handleConnectionClosed)
```

Comment:
This was removed because we don't want the connection to restart if the connection was intentionally closed.